apps/storybook/docs/g_recepten/events.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import {Meta} from '@storybook/addon-docs';
+
+<Meta title="Recepten/Events"/>
+
+# Events
```

Comment:
Ik zou er van maken

# Events

## Native- vs Custom-events

De UIG-componenten ontsluiten typisch native componenten die op hun beurt native events hebben. Web-componenten
[re-targetten](https://web.dev/articles/shadowdom-v1#the_shadow_dom_event_model) zelf bepaalde events, hierdoor lijken
die native events voor afnemers van de web-component zelf te komen. Daarnaast bieden de UIG-componenten eigen - custom -
events aan, deze hebben de prefix `vl-`. De prefix maakt duidelijk dat het UIG-events zijn en vermijden conflicten
met standaard events of events van andere bibliotheken. Op de documentatie pagina van een component vind je onder
`Configuratie` een overzicht van de custom-events, de native events worden niet vermeld!

Indien aanwezig heeft het de voorkeur te luisteren naar de custom events; deze worden voorzien bij componenten:
- die een complexe DOM-structuur hebben
- waar extra informatie meegegeven wordt met het event

Luister bvb. naar `vl-change` of `vl-input` (i.p.v. `click`) bij het aanvinken van `vl-checkbox-next`.


## Events in Lit

Lit, waar de nieuwere UIG-componenten gebruik van maken, heeft zelf een documentatie pagina over
[events](https://lit.dev/docs/v1/components/events).

Reply:
Aangepas zoals besproken